--- spnavcfg-0.3/Makefile.in.orig	2012-05-10 21:38:23.000000000 +0200
+++ spnavcfg-0.3/Makefile.in	2017-12-17 21:21:49.106368126 +0100
@@ -11,7 +11,7 @@
 LDFLAGS = `pkg-config --libs gtk+-2.0 gmodule-export-2.0` -lX11
 
 $(bin): $(obj)
-	$(CC) -o $@ $(obj) $(LDFLAGS)
+	$(CC) $(ldopt) -o $@ $(obj) $(LDFLAGS)
 
 -include $(dep)
 
@@ -32,13 +32,11 @@
 install:
 	$(INSTALL) -d $(DESTDIR)$(PREFIX)/bin
 	$(INSTALL) -m 4775 $(bin) $(DESTDIR)$(PREFIX)/bin/$(bin)
-	mkdir -p $(DESTDIR)$(PREFIX)/share/pixmaps
-	cp icons/spnavcfg-*.png $(DESTDIR)$(PREFIX)/share/pixmaps/
 	for i in 48 128 256; do \
 		destdir=$(DESTDIR)$(PREFIX)/share/icons/hicolor/$${i}x$${i}/apps; \
 		mkdir -p $$destdir; \
-		rm -f $$destdir/spnavcfg-$${i}x$${i}.png; \
-		ln -s $(DESTDIR)$(PREFIX)/share/pixmaps/spnavcfg-$${i}x$${i}.png $$destdir/spnavcfg-$${i}x$${i}.png; \
+		rm -f $$destdir/spnavcfg.png; \
+		cp -p icons/spnavcfg-$${i}x$${i}.png $$destdir/spnavcfg.png; \
 	done
 
 
